exercises/levenshtein-distance/canonical-data.json

```diff
@@ -0,0 +1,65 @@
+{
+  "exercise": "levenshtein-distance",
+  "version": "1.0.0",
+  "cases": [
+    {
+      "description": "Calculate Levenshtein distance",
+      "cases": [
+        {
+          "description": "mine_to_times",
+          "property": "distance",
+          "input": {
+            "phrase": "mine",
+			"phrase": "times"
+          },
+          "expected": "3"
```

Comment:
since the levenshtein distance is a number, let's use a number for the expected answer, not a string.